Automattic/wp-calypso
# Big Sky interstitial Modal

Superseding #91698 

Show an interstitial modal when selecting Big Sky.  

A couple of known to-do's: 
- [x] Choose a better icon for the "Give your creativity a head start"
- [x] Clean up spacing between elements 
- [x] Fire tracks event for closing the modal
- [x] Fire tracks event for continuing 
- [x] Fix console error about button nested in button

## Proposed Changes

Adds an interstitial modal when selecting Big Sky option, that will show a disclaimer. 

## Why are these changes being made?


*

## Testing Instructions



- Go through the /start/premium flow
- When you land on `/design-choices`, choose Big Sky (you can also go directly to `/setup/site-setup/design-choices?siteSlug=YOURSITE` if the site matches the criteria for Big Sky)
- The Modal will pop up
- Can close the modal with the X button
- Try in different browsers (styling)
- Clicking the "let's get started" button should successfully launch big sky (next step)

## Pre-merge Checklist
